src/test/groovy/net/researchgate/release/GitReleasePluginTests.groovy

```diff
@@ -130,7 +130,8 @@ class GitReleasePluginTests extends Specification {
         executor.exec(['git', 'checkout', 'myBranch'], failOnStderr: false, directory: remoteRepo, env: [:])
         executor.exec(['git', 'reset', '--hard', 'HEAD'], failOnStderr: false, directory: remoteRepo, env: [:])
         then:
-        newestCommit.contains("Signed-off-by: Unit Test <unit@test>")
+//        newestCommit.contains("Signed-off-by: Unit Test <unit@test>")
+        newestCommit.contains("[Gradle Release Plugin] - new version commit:  '1.1'")
```

Comment:
This renders the test useless. The point is, that it tests whether the `-s` is given to Git and thus the `Signed-off-by` line added.
Here this also works just fine.